docs/ConfigureScalarDL.md

```diff
@@ -2,156 +2,84 @@
 
 This guide explains how to configure Scalar DL custom values in helm charts for Scalar DL deployment.
 
-## Create Kubernetes Secrets
+## Update Configuration Files
 
-Kubernetes Secret is an object that contains a small amount of sensitive data such as a password, a token, or a key. 
-This optional method is highly recommended for handling credentials in the production environment.
-
-You need to create a secret object with `db-username` and `db-password` to store the username and password of the underlying database.
+### Kubernetes Secrets and Schema Loading reference
 
-```
-# Cosmos DB
-    # username=<Account name>
-    # password=<Cosmos DB account primary master key>
+You can refer to the table below to get values to create Kubernetes Secrets and update the [schema-loading-custom-values.yaml](../conf/schema-loading-custom-values.yaml) file according to the database you chose
 
-# DynamoDB
-    # username=<AWS_ACCESS_KEY_ID>
-    # password=<AWS_ACCESS_SECRET_KEY>
+| Database  | database | contactPoints              | username          | password                                      | dynamoBaseResourceUnit | cosmosBaseResourceUnit | 
```

Comment:
Added a table with configuration properties required for each database to create Kubernetes secret and deploy schema loading helm chart.